docs/usage/self-hosted-experimental.md

```diff
@@ -39,3 +39,8 @@ If set to "false" (string), Renovate will remove any existing `package-lock.json
 ## RENOVATE_USER_AGENT
 
 Configures the `user-agent` string to be sent with HTTP requests.
+
+## RENOVATE_X_HARD_EXIT
+
+If set to any value, Renovate will use a "hard" `process.exit()` once all work is done, even if a sub-process is otherwise delaying Node.js from exiting.
```

Comment:
Most of the time you need to use a specific number/string/boolean to get things configured properly.

Are you sure literally any value will work to enable `RENOVATE_X_HARD_EXIT`?
I think it's clearer to say: "Set to `x`", where `x` is a specific value.

Reply:
Any value will do - true, yes, y, no, never, HonkingGoose. Basically it just needs to be "set"

Reply:
Ah okay, we can keep the text as-is then. 😄